src/components/erc721/marketplace/dutch_auction_price_chart_card.tsx

```diff
@@ -95,30 +153,95 @@ export const DutchAuctionPriceChartCard = (props: Props) => {
     const { beginAmount, beginTimeSeconds } = getDutchAuctionData(makerAssetData);
     const price = getCollectiblePrice(collectible) as BigNumber;
     const { days, hours } = convertTimeInSecondsToDaysAndHours(expirationTimeSeconds.minus(beginTimeSeconds));
+
+    const currentPriceInUnits = +tokenAmountInUnits(price, ETH_DECIMALS);
+    const beginAmountInUnits = +tokenAmountInUnits(beginAmount, ETH_DECIMALS);
+    const endAmountInUnits = +tokenAmountInUnits(order.takerAssetAmount, ETH_DECIMALS);
+
+    const dataNumbers = generateMarkersByPrices({ beginAmountInUnits, endAmountInUnits, currentPriceInUnits });
+    const options = {
+        chart: {
+            type: 'area',
+            margin: [10, 5, 10, 10],
+            height: 148,
+            width: null,
+        },
+        title: {
+            text: '',
+        },
+        subtitle: {
+            text: '',
+        },
+        // tslint:disable
+        series: [
+            {
+                data: dataNumbers,
+                color: '#00AE99',
+                turboThreshold: dataNumbers.length,
+            } as Highcharts.SeriesColumnOptions,
+        ],
+        // tslint:enable
+        credits: {
+            enabled: false,
+        },
+        legend: {
+            enabled: false,
+        },
+        xAxis: {
+            title: {
+                text: '',
+            },
+            labels: {
+                enabled: false,
+            },
+            visible: false,
+        },
+        yAxis: {
+            title: {
+                text: '',
+            },
+            labels: {
+                enabled: false,
+            },
+            visible: false,
+            min: endAmountInUnits,
+            max: beginAmountInUnits,
+            startOnTick: false,
+            endOnTick: false,
+        },
+        tooltip: {
+            // tslint:disable
+            // prettier-ignore
+            formatter() {
+                const self: any = this;
+                return `Price: <b>${self.y} ETH</b>`;
+            }
+            // tslint:enable
+        },
+    };
+
     return (
         <Card>
             <CollectibleDescriptionInnerTitle>Price Chart</CollectibleDescriptionInnerTitle>
             <PriceChartContainer>
                 <PriceChartPriceAndTime>
                     <PriceChartTitle>Current Price</PriceChartTitle>
-                    <PriceChartValue>{tokenAmountInUnits(price, ETH_DECIMALS)} ETH</PriceChartValue>
+                    <PriceChartValue>{currentPriceInUnits} ETH</PriceChartValue>
                     <PriceChartTitle>Time Remaining</PriceChartTitle>
                     <PriceChartValue>{`${days} Days ${hours} Hrs`}</PriceChartValue>
                 </PriceChartPriceAndTime>
                 <PriceChartGraphWrapper>
-                    <PriceChartGraph />
+                    <PriceChartGraph>
+                        <HighchartsReact highcharts={Highcharts} options={options} />
```

Comment:
Req: can we add a file/component like `DecliningPriceGraph` that includes `generateMarkersByPrices`, `options` and everything that is kind of specific to Highcharts. I believe it should only be a presentational or functional component with a `Props` interface like [Paramprices](https://github.com/0xProject/0x-launch-kit-frontend/pull/412/files#diff-25b2a962c46b6793ffd7897381dc2cf1R89), so it can be used like:
```tsx
<DecliningPriceGraph beginAmountInUnits={beginAmountInUnits} endAmountInUnits={endAmountInUnits} currentPriceInUnits={currentPriceInUnits} />
```

The `dutch_auction_price_chart_card` doesn't need to know how the graph is implemented (and even more, the internal implementation of the graph may vary).

Reply:
Done!